include/aspect/particle/property/crystal_preferred_orientation.h

```diff
@@ -353,6 +354,7 @@ namespace aspect
            * @param data The particle data vector.
            * @param mineral_i The mineral to set the value of the rotation matrix of a grain for.
            * @param grain_i The grain to get the value of the rotation matrix of.
+           * @param rotation_matrix The rotation matrix to set for the grain in the mineral.
```

Comment:
unrelated, but rotation_matrix should be passed by const reference. Can you open a PR please?